CHANGELOG.md

```diff
@@ -37,6 +45,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Initial release.
 
 [unreleased]: https://github.com/heroku/buildpacks-deb-packages/compare/v0.0.3...HEAD
+
```

Comment:
stray whitespace (presume due to the broken IntelliJ formatter / markdown plugin?)

Reply:
yep. thought i had reconfigured it to prevent this but looks like setting the minimum blank lines to 0 doesn't work with definition elements 😞